graph/shared/src/lib/project-graph-data-service/fetch-project-graph-service.ts

```diff
@@ -73,14 +74,25 @@ export class FetchProjectGraphService implements ProjectGraphService {
   async getExpandedTaskInputs(
     taskId: string
   ): Promise<Record<string, string[]>> {
+    // Check cache first
+    if (this.expandedTaskInputsCache.has(taskId)) {
```

Comment:
is there some way for clearing the cache? If the project graph changes

Reply:
Remove the caching on this level and keep the one in `graph.ts` command directly. There's not a good place to invalidate this cache on the `graph-client` side.
- we don't currently have any polling for new task graph and because we're lazy loading task graph or project/target on demand, it's tricky to find a place to poll
- we definitely don't want to poll for expanded inputs